drivers/frequency/hmc7044/hmc7044.c

```diff
@@ -637,7 +646,9 @@ static int32_t hmc7044_setup(struct hmc7044_dev *dev)
 	mdelay(1);
 	hmc7044_write(dev, HMC7044_REG_REQ_MODE_0,
 		      (dev->high_performance_mode_clock_dist_en ?
-		       HMC7044_HIGH_PERF_DISTRIB_PATH : 0));
+		       HMC7044_HIGH_PERF_DISTRIB_PATH : 0) |
```

Comment:
This should be in a separate commit, with description, why it is needed.

Reply:
You are not using the latest version of no-OS - this has been removed by https://github.com/analogdevicesinc/no-OS/commit/1915ff0af51ee6b65470f6a4fa4a71bab9afa884

Reply:
Apologies, this snuck in due to a convoluted merge process on our end. Reverted.